src/realmGeol.ttl

```diff
@@ -57,307 +288,1624 @@ sorelsp:hasUpperBoundary rdf:type owl:ObjectProperty .
+                                rdfs:subClassOf soreag:GeologicBoundary ;
+                                rdfs:label "mohorovicic discontinuity"@en .
+
+
+###  http://sweetontology.net/realmGeol/Monocline
+soreag:Monocline rdf:type owl:Class ;
+                 rdfs:subClassOf soreag:StructuralFeature ;
+                 dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+                 dcterms:source <https://doi.org/10.1093/acref/9780199653065.001.0001> ;
+                 skos:definition "A one-limbed flexure on either side of which the strata are horizontal or dip uniformly at low angles. (Allaby, 2013)"@en ;
+                 skos:prefLabel "monocline"@en .
+
+
+###  http://sweetontology.net/realmGeol/MyloniteZone
+soreag:MyloniteZone rdf:type owl:Class ;
+                    rdfs:subClassOf soreag:StructuralFeature ;
+                    dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+                    dcterms:source <http://www.glossaryofgeology.org> ,
+                                   <https://doi.org/10.1093/acref/9780199653065.001.0001> ;
+                    skos:definition "Usually found in narrow, planar zones of localised ductile deformation, although kilometre-scale zones exist. Mylonites form in shear zones by mechanical grinding, crushing and recrystallization of rock to produce a metamorphic rock that has foliation and is much finer grained than its precursor. (Modified from: Neuendorf et al., 2011; Allaby, 2013)"@en ;
+                    skos:prefLabel "mylonite zone"@en .
+
+
+###  http://sweetontology.net/realmGeol/Nappe
+soreag:Nappe rdf:type owl:Class ;
+             rdfs:subClassOf soreag:TechtonostratigraphicEntity ;
+             dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+             dcterms:source <http://www.glossaryofgeology.org> ,
+                            <https://doi.org/10.1093/acref/9780199653065.001.0001> ;
+             skos:definition "A sheetlike, allochthonous rock unit, which has moved on a predominantly horizontal surface as a result of thrust faulting, recumbent folding, or both.  Fold limbs and axes are approximately horizontal (After Neuendorf et al., 2011; Allaby, 2013)"@en ;
+             skos:prefLabel "nappe"@en .
+
+
+###  http://sweetontology.net/realmGeol/NappeComplex
+soreag:NappeComplex rdf:type owl:Class ;
+                    rdfs:subClassOf soreag:TechtonostratigraphicEntity ;
+                    dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+                    dcterms:source <https://doi.org/10.1080/11035897.2016.1178666> ;
+                    skos:definition "A nappe complex consists of two or more distinct nappes that are stacked upon, and related to, one another in some significant respect, e.g., shared general environment of origin, similar metamorphism and/or deformation. (Kumpulainen, 2017)"@en ;
+                    skos:prefLabel "nappe complex"@en .
```

Comment:
I'd suggest that the restriction to 'widespread, if not worldwide' is unnecessary. Stratigraphic events can be observed in any stratigraphic section.  A different class for 'global' or 'widespread' stratigraphic events should be defined.

Deeper problem-- use of term 'event' will have the connotation of a temporal feature.  I think the intention here is that the 'stratigraphic event' is the physical manifestation of an event that occurred during deposition of a stratigraphic unit, so that 'stratigraphic event' is a physical thing that can be observed in outcrop.  In that understanding, then 'stratigraphic event' must be a kind of 'stratigraphic unit'.  

Given the restriction to 'widespread, if not worldwide' spatial extent, perhaps what is intended here is something like 'marker bed', which would be another kind of stratigraphic unit?